lib/pf/Switch/AeroHIVE/BR100.pm

```diff
@@ -0,0 +1,59 @@
+package pf::Switch::AeroHIVE::BR100;
+
+=head1 NAME
+
+pf::Switch::AeroHIVE::BR100
+
+=head1 SYNOPSIS
+
+Object oriented module to access and configure a AeroHIVE Branch Router 100.
+
+=head1 STATUS
+
+This module is currently only a placeholder, see pf::Switch::AeroHIVE
+
+=cut
+
+use strict;
+use warnings;
+use Log::Log4perl;
+use Net::SNMP;
+use Try::Tiny;
```

Comment:
would looks good that way:
use strict;
use warnings;

use Log::Log4perl;

use base ('pf::Switch::AeroHIVE');

use pf::Switch::constants;


sub supportsWiredMacAuth { return $TRUE; }